lib/ansible/executor/action_write_locks.py

```diff
@@ -15,30 +15,44 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-# Make coding more python3-ish
-from __future__ import (absolute_import, division, print_function)
+from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
+import sys
 from multiprocessing import Lock
 
 from ansible.module_utils.facts.system.pkg_mgr import PKG_MGRS
+from ansible.utils.multiprocessing import context as multiprocessing_context
+
+try:
+    # Do not initialize this more than once.
+    action_write_locks
+except (NameError, UnboundLocalError):
+    action_manager = multiprocessing_context.Manager()
+    if sys.version_info > (3, 0, 0):
```

Comment:
This doesn't look right. This basically allows Python 3.0.1+, not 3.0.0+ as you probably intended. But even with that, it's best to just compare the major version and use "py2 or not py2" because py3+ also means py4+.

OTOH we vendor `six` which contains proper constants already. So just import that:
```python
from ansible.module_utils import six

if six.PY2:
    ...
else:
    ...

```

Reply:
will do.